builtin/providers/aws/resource_aws_network_acl.go

```diff
@@ -397,7 +397,7 @@ func updateNetworkAclEntries(d *schema.ResourceData, entryType string, conn *ec2
 				}
 			}
 
-			if add.CidrBlock != nil {
+			if *add.CidrBlock != "" {
```

Comment:
Is this guaranteed to be set?  It looks optional:

- https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/network_acl_entry.go#L41-L43

In which case if it's optional then this will panic with a nil pointer deref, right?

Probably want `if add.CidrBlock != nil && *add.CidrBlock != ""{` ?